gulp/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+    "rules": {
+        "no-console": "off"
```

Comment:
ESLint allows per-directory configs, so this only applies to scripts in the `gulp` folder. I decided to allow `console` in Gulp scripts because 1) `console` is always defined in Node, whereas one of the reasons the `no-console` rule exists is IE9 doesn't implement `console`, and 2) `console.log` is pretty common in our gulp scripts as a means of feedback